### DISCUSSION TITLE
TIMELAPSE DE ARCHIVOS RASTER

### DISCUSSION THREAD

**Discussion by JamirGarro** [qgispe/discusiones#27] [Preguntas&Respuestas] [answered]
Hola, tengo 5 imágenes a las que realize el NDVI y existen diferencias, lo que busco es generar un timelapse para una presentación mas visual. he intentado con el plugin time manager de Qgis pero no logre obtener resultados, además los tutoriales que hay en internet por lo general solo lo hacen con capas vectoriales.
![imagen_2021-08-27_192856](https://user-images.githubusercontent.com/86505835/131200366-0afbe404-87c9-4d49-9d53-f0db4674f6b9.png)
si alguien tuviera alguna bibliografía para poder realizar esto en QGIS sería de mucha ayuda :)

**Comment by ambarja** [comment, marked as answer]
Hola @JamirGarro desde la versión __3.14__ está disponible la herramienta del  **Temporal Controller** dentro de QGIS , con esto puedes usar un TimeLapse tanto para datos raster como datos vectoriales, sin embargo si usas una versión menor puedes usar el plugin __Time Manager__, aquí te adjunta una imagen de referencia.
Si en caso deseas un mayor soporte no dudes en invitarme un café :coffee:  para charlar y ver ese tema.
Saludos!
![2021-08-31_21-59](https://user-images.githubusercontent.com/23284899/131604976-cfbfe543-d734-4b54-bc1c-82d3b074cfa5.png)
